src/base/static/components/templates/main-map.js

```diff
@@ -401,7 +402,11 @@ class MainMap extends Component {
 
   render() {
     return (
-      <Fragment>
+      <>
+        <InviteModal
+          router={this.props.router}
+          isOpen={this.props.isInviteModalOpen}
+        />
```

Comment:
I tried adding this modal in a way that would be unobtrusive to the AppView refactor. So I added it into MainMap instead of creating lots of changes in AppView.

Reply:
Thanks! This does make it easier.